--- a/node_modules/expo-widgets/ios/Widgets/WidgetLiveActivity.swift
+++ b/node_modules/expo-widgets/ios/Widgets/WidgetLiveActivity.swift
@@ -30,6 +30,7 @@ public struct WidgetLiveActivity: Widget {
         environment: environment
       )
       LiveActivityBannerView(context: context, nodes: nodes)
+        .widgetURL(getLiveActivityUrl(forName: context.state.name))
     } dynamicIsland: { context in
       let nodes = getLiveActivityNodes(
         forName: context.state.name,
